espnet2/bin/tts_inference.py

```diff
@@ -209,7 +211,9 @@ def __call__(
 
         # apply vocoder (mel-to-wav)
         if self.vocoder is not None:
-            if output_dict.get("feat_gen_denorm") is not None:
+            if self.prefer_normalized_feats and output_dict.get("feat_gen") is not None:
+                input_feat = output_dict["feat_gen"]
+            elif output_dict.get("feat_gen_denorm") is not None:
                 input_feat = output_dict["feat_gen_denorm"]
             else:
                 input_feat = output_dict["feat_gen"]
```

Comment:
Simplified if-else block.
```suggestion
            if self.prefer_normalized_feats or output_dict.get("feat_gen_denorm") is None:
                input_feat = output_dict["feat_gen"]
            else:
                input_feat = output_dict["feat_gen_denorm"]
```